README.md

```diff
@@ -18,7 +18,15 @@ Perform the following steps from your terminal.
     git clone https://github.com/datamade/bga-payroll.git
     cd bga-payroll
     ```
-2. Build and run the applicaton.
+2. Copy `bga_database/local-settings.example.py` to `bga_database/local-settings.py`.
+
+    ```bash
+    cp bga_database/local-settings.example.py bga_database/local-settings.py
+    ```
+
+3. Add `POSTGRES_PASSWORD=password` as an environment variable for the `postgres` service in `docker-compose.yml`.
```

Comment:
Any reason not to make this update in `docker-compose.yml`, instead of making it a step you have to do manually? We only use the main compose file for local development, so it's ok for the password to be weak/exposed.

Reply:
@hancush Much better idea, I added that here: 15bf5c0. But now I'm wondering if I should've added this to `docker-compose.deployment.yml` instead of `docker-compose.yml`?

Reply:
Great question. Let's leave the deployment compose file alone for now, since it's working in our deployment environments.